lib/trace_location/collector.rb

```diff
@@ -14,18 +13,23 @@ def collect(match:, ignore:, &block)
         hierarchy = 0
         id = 0
         cache = {}
+        method_source_cache = {}
 
         tracer = TracePoint.new(:call, :return) do |trace_point|
           next if match && !trace_point.path.to_s.match?(/#{Array(match).join('|')}/)
           next if ignore && trace_point.path.to_s.match?(/#{Array(ignore).join('|')}/)
 
           id += 1
-          caller_path, caller_lineno = trace_point.binding.of_caller(2).source_location
+          caller_loc = caller_locations(2, 1)[0]
```

Comment:
We can use `caller_locations` instead of `binding.of_caller`. It is faster, and no dependency.